src/sham/A26_ID_FromURL.java

```diff
@@ -0,0 +1,24 @@
+package sham;
+
+public class A26_ID_FromURL {
+
+	void findIdFromGivenURL()
+	{
+		String url = "https://automationbykrishna.com/name=techno/id=123";
+		String[] splitted=url.split("id=");
+		if(splitted[0].contains("123"))
```

Comment:
here if condition is not requires.. & also contains(123) is not correct way bcoz it might changes right.